msteams-platform/bots/how-to/conversations/conversation-messages.md

```diff
@@ -20,6 +20,16 @@ Your bot receives messages from Teams using the `Text` property and it sends sin
 
 For more information, see [user attribution for bot messages](/microsoftteams/platform/messaging-extensions/how-to/action-commands/respond-to-task-module-submit?tabs=dotnet%2Cdotnet-1#user-attribution-for-bots-messages).
 
+The following table lists the events that your bot can receive and take action on:
+
+| Event type | Payload object | Scope |
+| ---- | ---------| ------ |
+| [Receive a message](#receive-a-message) | `Activity` | All |
+| [Send a message](#send-a-message) | `SendActivityAsync` | All |
```

Comment:
Can we remove "send a message" from the table? Send a message is not an event that bot will receive. It is sent activity from bot.

Reply:
Updated the doc as you suggested.